shared/templates/sysctl/bash.template

```diff
@@ -3,7 +3,6 @@
 # strategy = disable
 # complexity = low
 # disruption = medium
-. /usr/share/scap-security-guide/remediation_functions
```

Comment:
This is the same, we need this because the `bash_instantiante_variables` contains a bash function call. :scream_cat: